ui/src/landing/CreateWish.test.js

```diff
@@ -32,18 +32,26 @@ describe('CreateWish tests', () => {
       wrapper.setState({
         name: 'a name',
         age: '14',
-        wishType: 'a type'
+        wishType: 'a type',
+        showChildInfo: false
       })
     })
 
+    it('Should hide Childinfo component', () => {
+      expect(wrapper.find(Childinfo).length).toEqual(0)
+    })
+
     it('Should show Childinfo component', () => {
+      wrapper.setState({
+        showChildInfo: true
+      })
```

Comment:
Something like `wrapper.instance().selectWishType("GO")` should resolve the codecov miss mentioned in `CreateWish.js`.